index.js

```diff
@@ -18,7 +18,7 @@ module.exports.plugins = [
   require("remark-lint-checkbox-content-indent"),
   [require("remark-lint-code-block-style"), "fenced"],
   require("remark-lint-definition-spacing"),
-  require("remark-lint-fenced-code-flag"),
+  [require("remark-lint-fenced-code-flag"), { allowEmpty: false }],
```

Comment:
According to the README for remark-lint-fenced-code-flag, the default value of `allowEmpty` is false. Either this is basically a no-op or I'm missing something.

Reply:
Oh, you're right! Sorry — I seem to have missed that bit. Interestingly, it also says the following.

> The object can have an array of `'flags'` which are allowed: other flags will not be allowed.

This might be the right plugin to supply an &ldquo;allowed list&rdquo; of info strings relevant for our subset of highlight.js grammars. I was hoping to be able to provide suggestions to users though, so not sure if this will be right fit.

Thoughts?

Reply:
Seems worth a shot. Start with every string we have now, and then open PRs removing ones we don't want one at a time? Like, maybe change all `txt` to `text` first (or vice versa)?

Reply:
That sounds like a good plan. Seeing as how the branch of @zeke's PR (https://github.com/nodejs/node/pull/33028) is `normalize-plaintext-code-blocks`, I think we should keep that one scoped specifically to normalizing Plaintext grammars and avoid Shell Session grammars (`console`). I will have to do a more thorough audit to ensure we get all of them in one shot — will follow up in that PR thread.

Drafting this PR until we have a suitable replacement.